### ISSUE TITLE
Review Summary - Change to a «POST -> redirect -> GET cycle»

### ISSUE THREAD

**Issue by faneca** [fabd/kanji-koohii#51]
Maybe it's a matter of opinion, but I prefer to avoid seeing *expired pages* when pressing the back button on the browser. In the Review Summary case, going to the second page of review statistics and then pressing the back button won't work as desired.

**Comment by fabd** [comment]
I agree it's not ideal also the table paging is supposed to be AJAX and not a GET request (it is ajax on some pages). 

Tagging as a bug, and UX Improvement category.
